operations/helm/charts/mimir-distributed/templates/tokengen/tokengen-job.yaml

```diff
@@ -65,32 +69,60 @@ spec:
           {{- if or .Values.global.extraEnv .Values.tokengenJob.env }}
           env:
             {{- with .Values.global.extraEnv }}
-              {{ toYaml . | nindent 12 }}
+            {{- toYaml . | nindent 12 }}
             {{- end }}
             {{- with .Values.tokengenJob.env }}
-              {{ toYaml . | nindent 12 }}
+            {{- toYaml . | nindent 12 }}
             {{- end }}
           {{- end }}
           {{- if or .Values.global.extraEnvFrom .Values.tokengenJob.extraEnvFrom }}
           envFrom:
             {{- with .Values.global.extraEnvFrom }}
-              {{- toYaml . | nindent 12 }}
+            {{- toYaml . | nindent 12 }}
             {{- end }}
             {{- with .Values.tokengenJob.extraEnvFrom }}
-              {{- toYaml . | nindent 12 }}
+            {{- toYaml . | nindent 12 }}
             {{- end }}
           {{- end }}
           securityContext:
             {{- toYaml .Values.tokengenJob.containerSecurityContext | nindent 12 }}
+      containers:
+        - name: kubectl
+          image: {{ include "mimir.kubectlImage" . }}
+          imagePullPolicy: {{ .Values.kubectlImage.pullPolicy }}
+          command:
+            - "/bin/sh"
+            - "-c"
+            - |
+              if [ -f /shared/admin-token ]; then
+                echo "Admin token generated successfully"
+                {{- if .Values.tokengenJob.storeTokenInSecret | default true }}
```

Comment:
isn't this `true` by default already due to the values.yaml? why do we need the `|default true`?

Reply:
removed